ietf/doc/urls.py

```diff
@@ -82,14 +82,14 @@
 
 # End of block that should be an idealized docs.ietf.org service instead
 
-    url(r'^html/(?P<name>[Rr][Ff][Cc] [0-9]+?)(\.txt|\.html)?/?$', views_doc.document_html),
+    url(r'^html/(?P<name>[Rr][Ff][Cc]\s*[0-9]+?)(\.txt|\.html)?/?$', views_doc.document_html),
```

Comment:
What is the problem that led to wanting to do this? It changes the contract for what we'll provide, and we would be expected to keep this working in perpetuity. 
If it's a linkifier figuring out what to do, why not make it the linkifiers problem to normalize the produced URL instead?